.gitignore

```diff
@@ -27,6 +27,10 @@ var/
 *.egg
 build-deb/
 _version.py
+venv*/
```

Comment:
I attempted such change 2 weeks ago, but it was discouraged, see https://github.com/christiansandberg/canopen/pull/468/files/5c2343afd7af3e225d1101285d0562b7969b46e0#r1648161548

Reply:
That's too bad. Might as well just close this, then 🤷

Reply:
Well, the `docs` fix is still relevant.